File: src/main/java/me/arasple/mc/trmenu/action/acts/ActionConnect.java
package me.arasple.mc.trmenu.action.acts;

import me.arasple.mc.trmenu.action.base.AbstractAction;
import me.arasple.mc.trmenu.utils.Bungees;
import me.arasple.mc.trmenu.utils.Vars;
import org.bukkit.entity.Player;

/**
 * @author Arasple
 * @date 2019/12/28 19:07
 */
public class ActionConnect extends AbstractAction {
    @Override
    public String getName() {
        return "connect|bungee|server";
    }

    @Override
    public void onExecute(Player player) {
        Bungees.connect(player, getContent(player));
    }
}

File: src/main/kotlin/me/arasple/mc/trmenu/modules/action/impl/ActionConnect.kt
package me.arasple.mc.trmenu.modules.action.impl

import me.arasple.mc.trmenu.modules.action.base.Action
import me.arasple.mc.trmenu.utils.Bungees
import org.bukkit.entity.Player

/**
 * @author Arasple
 * @date 2020/3/8 21:18
 */
class ActionConnect : Action("bungee|server|connect") {

    override fun onExecute(player: Player) = Bungees.connect(player, getContent(player))

}